### PR TITLE
Also detect plugins during development

### LINKED ISSUE #77: Plugin not loaded by Pest on development
While creating a new plugin for Pest it seams that the plugin could not be loaded while on the development phase.

Me and @nunomaduro found that the reason was because the "Plugin::class" was not being added on the `vendor/pest-plugins.json` file … Even that all the required setup for the plugin was properly defined.

### PR DESCRIPTION
**Pull request by fetzi**
fixes pestphp/pest#77

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
     "type": "composer-plugin",
     "require": {
         "php": "^7.3",
-        "composer-plugin-api": "^1.1||^2.0"
+        "composer-plugin-api": "^1.1||^2.0",
+        "pestphp/pest": "^0.2"
```
Comment (nunomaduro) [inline composer.json:19]: Everything sounds good. Just not sure about this on as it may cause circular dependencies. I will check it tonight. Thanks!

Comment (fetzi) [reply composer.json:19]: I think we need the dependency because we are using the `Container` class.

Comment (nunomaduro) [reply composer.json:19]: Well, yeah - but pest depends on this plugin too. And composer will go crazy with circular dependencies.

I will check tonight. ( or try )

**Comment by nunomaduro** [comment]
Thanks!

**Comment by nunomaduro** [comment]
`v0.2.1` contains this feature

**Pull request merged**